nixos/modules/services/security/esdm.nix

```diff
@@ -0,0 +1,132 @@
+{ lib, config, pkgs, ... }:
+
+let
+  cfg = config.services.esdm;
+  kernelVersion = config.boot.kernelPackages.kernel.version;
+  linux_6_3 = (builtins.compareVersions kernelVersion "6.3" >= 0);
```

Comment:
```suggestion
  linux_6_3 = (lib.versionAtLeast kernelVersion "6.3");
```

Reply:
Yeah that looks a bit nicer. Incorporated it in commit: 1f5bc3d821bf1d1d80b654a2eb29080335f7d6be.